pom.xml

```diff
@@ -832,6 +848,74 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <!-- Serves as support for configuring Revapi -->
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>parse-version</id>
+                        <goals>
+                            <!-- This defines the ${parsedVersion.*} properties used in the Revapi config. -->
+                            <goal>parse-version</goal>
+                        </goals>
+                        <phase>validate</phase>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.revapi</groupId>
+                <artifactId>revapi-maven-plugin</artifactId>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.revapi</groupId>
+                        <artifactId>revapi-java</artifactId>
+                        <version>${version.revapi-java.plugin}</version>
+                    </dependency>
+                </dependencies>
+                <configuration>
+                    <failOnMissingConfigurationFiles>false</failOnMissingConfigurationFiles>
+                    <!-- Consider changes from the latest .Final version, not from the latest non-snapshot. -->
+                    <versionFormat>\d+\.\d+\.\d+\.Final</versionFormat>
+                    <ignoreSuggestionsFormat>xml</ignoreSuggestionsFormat>
+                    <analysisConfiguration>
+                        <revapi.semver.ignore>
+                            <!-- Automatically ignore changes that are OK according to the semver rules. -->
+                            <enabled>true</enabled>
+                        </revapi.semver.ignore>
+                        <revapi.java.filter.annotated>
+                            <exclude>
+                                <!-- Don't break on changes in the incubating API. -->
+                                <item>@io.debezium.common.annotation.Incubating</item>
+                            </exclude>
+                        </revapi.java.filter.annotated>
+                    </analysisConfiguration>
+                    <analysisConfigurationFiles>
+                        <configurationFile>
+                            <!--
+                                Each API checked module can have a file detailing the intentional API changes
+                                in the form of a configuration for Revapi.
+                            -->
+                            <path>${basedir}/src/chore/intentional-api-changes.xml</path>
```

Comment:
What does "chore" mean?

Reply:
It's a convention used by some projects for keeping support files for "house chores" - e.g. things that are not code itself but are "around it". Feel free to suggest some other location that would fit better with your conventions.

Reply:
Aaah, gotcha. So I think the closest in Debezium would be a new sub-dir under _support_.

Reply:
Hmm... as far as I understand the support/* directories - these essentially are project wide support modules - e.g. they can be used by all modules. On the other hand, these lists of API changes are specific to each module.

While I theoretically could create a new module in support and there, for example, create a separate file for each module's API changes, for me that would feel "far" (in terms of directory structure) from the actual code changes. However, if that's the convention you're used to that's no problem, just pointing it out...

Reply:
Well, I think I spoke too soon... Since you have separate package for each module and you version all modules uniformly, just a single file under support most probably would work.

Reply:
Yes, I was wondering that, too. It could be done either way, but I think for now I'd prefer to have it all in a new module under _support_.

Reply:
Ah, even better then!